packages/manager/modules/pci/src/projects/project/storages/databases/database/logs/logs.html

```diff
@@ -0,0 +1,31 @@
+<div class="container-fluid px-0 mt-3">
+    <!-- error messages -->
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <h2 data-translate="pci_database_logs_title"></h2>
+
+    <p data-translate="pci_databases_logs_description"></p>
+    <oui-field
+        label="{{:: 'pci_databases_logs_autorefresh' | translate }}"
+        class="flex-right"
+    >
+        <oui-switch
+            model="$ctrl.pollLogs"
+            name="oui-switch"
+            on-change="$ctrl.changePoll(modelValue)"
+        >
+        </oui-switch>
+    </oui-field>
+
+    <tail-logs>
+        <p
+            ng-repeat="line in $ctrl.logs track by $index"
+            ng-class="$ctrl.findMessageLevel(line.message)"
```

Comment:
```suggestion
            ng-class="$ctrl.getMessageLevelClassNames(line.message)"
```
for better understandability.

Reply:
Done with https://github.com/ovh/manager/commit/f63f3039f09d8eb53a20f7286e6f06666b538609